jdk/src/macosx/native/sun/font/AWTStrike.m

```diff
@@ -107,7 +107,7 @@ + (AWTStrike *) awtStrikeForFont:(AWTFont *)awtFont
 #define AWT_FONT_CLEANUP_FINISH                                         \
     if (_fontThrowJavaException == YES) {                               \
         char s[512];                                                    \
-        sprintf(s, "%s-%s:%d", THIS_FILE, __FUNCTION__, __LINE__);       \
+        snprintf(s, sizeof(s), "%s-%s:%d", __FILE__, __FUNCTION__, __LINE__);       \
```

Comment:
For consistency, I don't think the THIS_FILE change should be made. It was removed on the way to jdk11 by OpenJDK 8226346 (https:/github.com/ibmruntimes/openj9-openjdk-jdk11/commit/8fab614110), this OpenJDK issue isn't public. When THIS_FILE was removed, it was done as part of a bigger change to a number of files and not just this one. 

https:/github.com/ibmruntimes/openj9-openjdk-jdk11/commit/8fab614110

Reply:
Fixed as suggested.